src/main/resources/application.properties

```diff
@@ -1,6 +1,6 @@
 spring.datasource.url=jdbc:mysql://localhost:3306/testdb
 spring.datasource.username=root
-spring.datasource.password=admin
```

Comment:
For now if you could change this back to admin so it doesn't break everyone else's code that would be great. I'll look into a better solution for this in the near future.

Reply:
Ah yes, I forgot to bring up in the meeting and meant to change it back for the commit/push. I have been unable to change my root account password.. I've followed a few tutorials and get no error, but no results either. Perhaps we can use a non-root user?